packages/playwright/src/matchers/toMatchAriaSnapshot.ts

```diff
@@ -75,10 +89,10 @@ export async function toMatchAriaSnapshot(
     }
   };
 
-  let suggestedRebaseline: string | undefined;
-  if (!this.isNot && pass === this.isNot) {
-    if (updateSnapshots === 'all' || (updateSnapshots === 'missing' && !expected.trim()))
-      suggestedRebaseline = `toMatchAriaSnapshot(\`\n${unshift(received, '${indent}  ')}\n\${indent}\`)`;
+  if (!this.isNot && pass === this.isNot && generateNewBaseline) {
+    // Only rebaseline failed snapshots.
+    const suggestedRebaseline = `toMatchAriaSnapshot(\`\n${indent(received, '${indent}  ')}\n\${indent}\`)`;
+    return { pass: this.isNot, message: () => '', name: 'toMatchAriaSnapshot', suggestedRebaseline };
```

Comment:
```suggestion
    return { pass, message: () => '', name: 'toMatchAriaSnapshot', suggestedRebaseline };
```

Reply:
I want to always say pass: this.isNot to stringly suggest this is a failure